app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalController.php

```diff
@@ -48,9 +49,16 @@ class KnowledgeManagementPortalController extends Controller
     public function show(): JsonResponse
     {
         $settings = resolve(PortalSettings::class);
+        $settingsProperty = SettingsProperty::getInstance('portal.logo');
```

Comment:
Is there a particular reason we are  calling the `SettingsProperty` class here instead of putting from `$settings` above?

Reply:
@Orrison Actually I took refrenece from logo.blade file
![Screenshot 2024-06-06 231540](https://github.com/canyongbs/aidingapp/assets/170343024/7863d187-5e5f-49a3-b6a2-30cb5b25f0ab)

Reply:
Got it, if you can try pulling from the `$settings` variable already set here and see if that works.

Reply:
@Orrison I tried but that gave error.

Reply:
Okay We can leave this for now then